app/serializers/trip_searcher_serializer.rb

```diff
@@ -0,0 +1,5 @@
+class TripSearcherSerializer < ApplicationSerializer
```

Comment:
We could introduce a base SearcherSerializer and inherit from that for Trips and Stops. They both would then have the exact same APIs.